django-stubs/contrib/sites/managers.pyi

```diff
@@ -1,9 +1,8 @@
 from typing import TypeVar
 
-from django.contrib.sites.models import Site
 from django.db import models
 
-_T = TypeVar("_T", bound=Site)
+_T = TypeVar("_T", bound=models.Model)
```

Comment:
Looks like my comment was not sent the first time :(
This feels unrelated?

Reply:
Yeah it is. But a test on it errored on the `ClassVar` thingy and I realised that it was incorrectly typed. So I just corrected it.

Reply:
Can be found in #2308